config/templates/signup-invite.html

```
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title></title>
```

HTMLHint (1.9.2)

[error] <title></title> must not be empty.

(title-require)